procedures/procedure3.sql

```diff
@@ -0,0 +1,26 @@
+-- Database Function - Book Borrowing Frequency:
+-- Function Name: fn_BookBorrowingFrequency
+-- Purpose: Gauge the borrowing frequency of a book.
+-- Parameter: BookID
+-- Implementation: Count the number of times the book has been issued.
+-- Return: Borrowing count of the book.
+
+USE [LibraryManagement]
+GO
+
+CREATE FUNCTION dbo.fn_BookBorrowingFrequency
+(
+    @BookID numeric(18, 0) 
+)
+RETURNS int 
```

Comment:
Return early when input is null/invalid
`RETURNS INT 
WITH RETURNS NULL ON NULL INPUT`